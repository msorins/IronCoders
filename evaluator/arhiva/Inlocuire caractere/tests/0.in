1 2
Ana are mere mere mere da da da sigur dac.
da nuu
mere pere
