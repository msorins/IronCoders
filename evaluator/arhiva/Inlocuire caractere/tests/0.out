Ana are pere pere pere nuu nuu nuu sigur dac.